modules/social_features/social_group/modules/social_group_request/config/install/core.entity_view_display.group_content.group_content_type_7fcb76fdf61a9.default.yml

```diff
@@ -0,0 +1,33 @@
+langcode: en
+status: true
+dependencies:
+  config:
```

Comment:
please add enforced module dependencies so we can easily reinstall this module, now uninstall and reinstalling it gives a:

`Drupal\Core\Config\PreExistingConfigException: Configuration objects (core.entity_view_display.group_content.group_content_type_7fcb76fdf61a9.default, field.field.group_content.group_content_type_7fcb76fdf61a9.grequest_status,                                      [error]
field.field.group_content.group_content_type_7fcb76fdf61a9.grequest_updated_by, field.field.message.approve_request_join_group.field_message_context, field.field.message.approve_request_join_group.field_message_destination,
field.field.message.approve_request_join_group.field_message_related_object, field.field.message.request_to_join_a_group.field_message_context, field.field.message.request_to_join_a_group.field_message_destination,
field.field.message.request_to_join_a_group.field_message_related_object, field.storage.group_content.field_grequest_message, group.content_type.group_content_type_7fcb76fdf61a9, message.template.approve_request_join_group,
message.template.request_to_join_a_group) provided by social_group_request already exist in active configuration in /app/web/core/lib/Drupal/Core/Config/PreExistingConfigException.php:65
`